tests/performance/src/test/resources/scatter-gather-perf-test.xml

```diff
@@ -22,15 +22,6 @@
         </scatter-gather>
     </flow>
 
-    <flow name="sequentialProcessing">
```

Comment:
Isn't there another wy to do sequential processing now, to migrate this test to use?